### PR TITLE
UX: Improve styling of sidebar on mobile

### PR DESCRIPTION
**Pull request by tgxworld**
* Make sidebar footer is sticky
* Ensure that only the sidebar sections is scrollable

### Before

Screenshot from 2022-09-14 12-29-35

### After

Peek 2022-09-14 12-29

### CODE REVIEW

File: app/assets/stylesheets/common/base/sidebar.scss [hunk 1]
```diff
@@ -45,10 +45,6 @@
     box-sizing: border-box;
     flex: 1;
     padding: 1em 0;
-    box-sizing: border-box;
-    flex: 1;
-    display: flex;
-    flex-direction: column;
```
Comment (tgxworld) [inline app/assets/stylesheets/common/base/sidebar.scss:51]: Duplication

File: app/assets/stylesheets/mobile/menu-panel.scss [hunk 2]
```diff
@@ -61,19 +62,31 @@
 
 .panel-body-contents {
   // 2em padding very useful for iOS Safari's overlayed bottom nav
-  padding-bottom: calc(env(safe-area-inset-bottom) + 2em);
+  // padding-bottom: calc(env(safe-area-inset-bottom) + 2em);
```
Comment (pmusaraj) [inline app/assets/stylesheets/mobile/menu-panel.scss:65]: Did you want to remove this?

**Review by pmusaraj — APPROVED** [review]

File: app/assets/stylesheets/mobile/menu-panel.scss (hunk 2 above)
Comment (tgxworld) [reply app/assets/stylesheets/mobile/menu-panel.scss:65]: Oops that should be kept.

**Comment by tgxworld** [comment]
Thank you for reviewing @pmusaraj

**Pull request merged**